app/wallet/vsp.js

```diff
@@ -93,8 +94,16 @@ export const getAllVSPs = withLogNoData(
   "getAllVspsInfo"
 );
 
-// allowVSPHost enables the external request to a specif VSP host.
+// allowVSPHost enables the external request to a specific VSP host.
 export const allowVSPHost = log(
-  (host) => Promise.resolve(ipcRenderer.sendSync("allow-vsp-host", host)),
-  "Allow StakePool Host"
+  async (host) => {
+    // TODO: only ask for confirmation if VSP host is not yet allowed.
```

Comment:
This needs to be fixed before merging (this is only being left as-is to ease testing).